reactor-netty-core/src/main/java/reactor/netty/transport/ServerTransportConfig.java

```diff
@@ -161,7 +160,12 @@ protected ServerTransportConfig(ServerTransportConfig<CONF> parent) {
 
 	@Override
 	protected Class<? extends Channel> channelType(boolean isDomainSocket) {
-		return isDomainSocket ? ServerDomainSocketChannel.class : ServerSocketChannel.class;
+		throw new UnsupportedOperationException();
+	}
+
+	@Override
+	protected ChannelFactory<? extends Channel> connectionFactory(boolean isDomainSocket) {
+		throw new UnsupportedOperationException();
 	}
```

Comment:
are these UnsupportedException temporary ? I mean: is there a TODO in order to fix these UnsupportedOperationException ?

Reply:
No `ServerTransportConfig` cannot support `SocketChannel`. See `channelType` javadoc.

Reply:
ok.